google_places.py

```diff
@@ -40,8 +41,8 @@ def get_location():
     # {u'city': u'Hyderabad', u'longitude': 78.4744,
     # u'latitude': 17.3753, u'state': u'Telangana', u'IPv4': u'157.48.48.45',
     #  u'country_code': u'IN', u'country_name': u'India', u'postal': u'500025'}
-    url = "https://geolocation-db.com/json"
-    response = urllib.urlopen(url)
+    url = config.geolocation_api
+    response = urllib.request.urlopen(url) #outra fix-urllib.urlopen
```

Comment:
You have already created separate merge request for this. So you can exclude this :).